.env

```diff
@@ -51,4 +51,6 @@ CKAN__HARVEST__LOG_LEVEL=info
 CKAN__HARVEST__LOG_SCOPE=0
 
 CKANEXT__GEODATAGOV__BUREAU_CSV__URL=https://project-open-data.cio.gov/data/omb_bureau_codes.csv
-CKANEXT__GEODATAGOV__BUREAU_CSV__URL_DEFAULT=https://project-open-data.cio.gov/data/omb_bureau_codes.csv
\ No newline at end of file
+CKANEXT__GEODATAGOV__BUREAU_CSV__URL_DEFAULT=https://project-open-data.cio.gov/data/omb_bureau_codes.csv
+
+CKAN__SPATIAL__SRID=4326
```

Comment:
Just out of curiosity, what is SRID?

Reply:
Is about which _map prjection_ you use to save spatial data. [More info](https://en.wikipedia.org/wiki/Spatial_reference_system)